components/pipeline-service/kustomization.yaml

```diff
@@ -2,14 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- openshift-pipelines
-- pipelines-as-code
-- pvc-cleaner
-- tekton-chains
-- tekton-results
-
-# Skip applying the Tekton operands while the Tekton operator is being installed.
-# See more information about this option, here:
-# https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types
-commonAnnotations:
-  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+- git::https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=new-main
```

Comment:
https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/bootstrap-cluster.sh#L75-L100 should be updated, it's using tekton-results namespace, the cert is probably not needed anymore if CertManager is used.

Reply:
That's true for tekton-results-tls where we are creating a cert via certManager but currently the credentials for the postgres DB are not available as a manifest (rather generated via scripts like it is the case with the preview hack). I will see this can be handled via a manifest without a hardcoded pass